source/_includes/code_example.liquid

```diff
@@ -0,0 +1,81 @@
+{% assign first_tab_id = ui_id %}
+{% assign second_tab_id = first_tab_id | plus: 1 %}
+{% assign third_tab_id = second_tab_id | plus: 1 %}
```

Comment:
Instead of using incrementing IDs that we have to pass in from the place this is used, what about having this template itself auto-generate a unique ID, and then internally increment it? I'm not sure why the outer template needs to care about this, and I also don't think it's very important to have them easily readable and incremented. Is that a requirement of the jQuery UI widget?